R/FilterState-utils.R

```diff
@@ -19,7 +19,38 @@
 #'
 #' @param ... additional arguments to be saved as a list in `private$extras` field
 #'
-#' @seealso examples found here: `vignette("internal_function_examples", package = "teal.slice")`.
+#' @examples
+#' # use non-exported function from teal.slice
+#' include_js_files <- getFromNamespace("include_js_files", "teal.slice")
+#' init_filter_state <- getFromNamespace("init_filter_state", "teal.slice")
+#'
+#' filter_state <- init_filter_state(
+#'   x = c(1:10, NA, Inf),
+#'   x_reactive = reactive(c(1:10, NA, Inf)),
+#'   slice = teal_slice(
+#'     varname = "x",
+#'     dataname = "dataname"
+#'   ),
+#'   extract_type = "matrix"
+#' )
+#'
+#' shiny::isolate(filter_state$get_call())
```

Comment:
Since we don't prefix `shinyApp` and `shiny` is attached anyway, let's remove this as well.
```suggestion
#' isolate(filter_state$get_call())
```